### ISSUE TITLE
Link for the <figcaption>

### ISSUE THREAD

**Issue by EdMueller** [jonathanheilmann/ext-jh_magnificpopup#61]
Inside the file
> ext-jh_magnificpopup/Resources/Private/Partials/Magnificpopup/ContentTemplates/File/FluidStyledContent.html

a link is created for the image:
```
<div class="ce-textpic ce-{imageorient} ce-above">
  <div class="ce-gallery" data-ce-columns="1" data-ce-images="1">
    <f:if condition="{imageorient} == 'center'"><div class="ce-outer"><div class="ce-inner"></f:if>
    <div class="ce-row">
      <div class="ce-column">
        <figure class="image">
         {tsLink->f:format.raw()}
         <f:if condition="{imagecaption}">
          <figcaption>{imagecaption}</figcaption>
         </f:if>
        </figure>
      </div>
    </div>
    <f:if condition="{imageorient} == 'center'"></div></div></f:if>
  </div>
</div>
```
But i need that link for the "figcaption", too. Is that possible? Or around the complete "figure" tag?

**Comment by jonathanheilmann** [comment]
Actually version 1.0.x does not support this.

But as it's a eligible suggestion, I've added this to next version 1.2.0. 

Actually this version is under development within branch [https://github.com/jonathanheilmann/ext-jh_magnificpopup/tree/v1.1.0](v1.1.0), extension as zip: [jh_magnificpopup_1.1.0-dev_201807072235.zip](https://github.com/jonathanheilmann/ext-jh_magnificpopup/files/2173071/jh_magnificpopup_1.1.0-dev_201807072235.zip)
To enable linked image caption, please go to constant editor and set `plugin.tx_jhmagnificpopup.linkImagecaption` in `plugin.tx_jhmagnificpopup` section `general`.

**Comment by EdMueller** [comment]
Oh, thank you for the quick answer. I tried to fix that in version 1.0.1 with javascript
```
$(".tx-jhmagnificpopup-pi1 .ce-gallery figure").on ("click", "figcaption", function () {
  $ (this).siblings ().trigger ("click");
});
```
and it works :-). BUT: After closing the popup, the viewport jumps to the top of the page. Could i prevent this behavior? Could i send you a pm with live example?

**Comment by jonathanheilmann** [comment]
I'd recommend a native solution as provided in version 1.1.0-dev.
But you can send me a link to mail@jonathan-heilmann.de and I'll have a look at your example.

**Comment by jonathanheilmann** [comment]
Resolved within version 1.1.0
